app/views/rating_questions/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<div class="form"><%= form_with model: @rating_question, url: rating_question_path,local: true do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :tag %>
+  <%= f.text_field :tag %>
+  <%= f.submit %>
+  <% end %>
+  <%= link_to 'Back', rating_questions_path %>
+  <div class="flash-notice"><%= flash[:notice] %>
```

Comment:
Put this flash notice element in the application layout, rather than repeating it in each view.

Reply:
good idea